src/solvers/dg_unstructured_quad/containers_2d.jl

```diff
@@ -195,16 +195,16 @@ function init_interfaces!(interfaces, edge_information, boundary_names, polydeg,
       primary_element = edge_information[3,j]
       # Note: This is a way to get the neighbour element number and local side from a square
       #       structured mesh where the element local surface numbering is right-handed
-      if boundary_names[primary_side, primary_element] == "Bottom"
+      if boundary_names[primary_side, primary_element] === :Bottom
```

Comment:
Nice, I was about to comment that this should fix the broken periodic test. To do symbol comparison one uses three `===` instead of two `==`? Is this because we are actually checking for equivalence instead of equality

Reply:
You can also use `==`, but `===` is slightly faster for `Symbol`s:
```julia
julia> using BenchmarkTools

julia> foo(x) = x == :test
foo (generic function with 1 method)

julia> bar(x) = x === :test
bar (generic function with 1 method)

julia> @benchmark foo($(Ref(:test123))[])
BenchmarkTools.Trial: 
  memory estimate:  0 bytes
  allocs estimate:  0
  --------------
  minimum time:     1.264 ns (0.00% GC)
  median time:      1.266 ns (0.00% GC)
  mean time:        1.270 ns (0.00% GC)
  maximum time:     4.514 ns (0.00% GC)
  --------------
  samples:          10000
  evals/sample:     1000

julia> @benchmark bar($(Ref(:test123))[])
BenchmarkTools.Trial: 
  memory estimate:  0 bytes
  allocs estimate:  0
  --------------
  minimum time:     1.057 ns (0.00% GC)
  median time:      1.061 ns (0.00% GC)
  mean time:        1.072 ns (0.00% GC)
  maximum time:     16.334 ns (0.00% GC)
  --------------
  samples:          10000
  evals/sample:     1000

julia> @benchmark foo($(Ref(:test))[])
BenchmarkTools.Trial: 
  memory estimate:  0 bytes
  allocs estimate:  0
  --------------
  minimum time:     1.264 ns (0.00% GC)
  median time:      1.267 ns (0.00% GC)
  mean time:        1.272 ns (0.00% GC)
  maximum time:     15.878 ns (0.00% GC)
  --------------
  samples:          10000
  evals/sample:     1000

julia> @benchmark bar($(Ref(:test))[])
BenchmarkTools.Trial: 
  memory estimate:  0 bytes
  allocs estimate:  0
  --------------
  minimum time:     1.057 ns (0.00% GC)
  median time:      1.060 ns (0.00% GC)
  mean time:        1.074 ns (0.00% GC)
  maximum time:     16.020 ns (0.00% GC)
  --------------
  samples:          10000
  evals/sample:     1000
```